server/playground/index.js

```diff
@@ -0,0 +1,33 @@
+const buildSchema = require("@aerogear/data-sync-gql-core");
+const { makeExecutableSchema } = require("graphql-tools");
+const { ApolloServer } = require("apollo-server-express");
+const { log } = require("../logger");
+const models = require("../models");
+
+async function connectDataSources(dataSources) {
+    log.info("Connecting data sources");
+    Object.keys(dataSources).forEach(async key => {
```

Comment:
I think nested async functions do not work great and are source of bugs.
Could we replace that with `for of` loop that do not require function?